src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -77,6 +77,8 @@
 
 _AZ_LOGIN_MESSAGE = "Please run 'az login' to setup account."
 
+_USE_VENDORED_SUBSCRIPTION_SDK = True
```

Comment:
Do we need to change this to `False` in the future ?

Reply:
Probably not. `azure-cli-core` uses only very limited functionalities from this SDK, so we can always use the fixed version.